react-native.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  project: {
+    ios: {
+      sharedLibraries: [
+        "libz"
+      ]
+    },
+    android: {
```

Comment:
Minor question, but any reason [`android`, `ios`] are no longer in alphabetical order (like in the original file)?

Reply:
I understand it does not matter from a program execution point of view. It matters from a code cleanliness, maintenance, VCS point of view (the order has been changed from the original code)

Reply:
ah I see